server/apps/main/templates/main/signup.html

```diff
@@ -0,0 +1,125 @@
+            margin-bottom: 0;
+        }
+
+        ul {
+            list-style-type: none;
+            width: 100%;
+            display: flex;
+            justify-content: center;
+        }
+        .nav-item {
+            padding: 1.5em 2em 1.5em 2em;
+            font-size: large;
+            font-weight: bold;
+        }
+        .nav-link {
+            color: black;
+        }
+
+        .nav-link:hover {
+            text-decoration: none;
+            color: black;
+        }
+        #project-sign-up {
+            margin-left: 2em;
+            font-size: large;
+            margin-top: 0.5em;
+        }
+        .container {
+            margin: auto;
+        }
+        #form-check {
+            display: flex;
+            /* height: 50vh; */
+            padding-top: 25vh;
+
+        }
+        .footer {
+            background-color: gray;
+            height: 10vh;
+        }
+        .submit {
+            margin-left: 45%;
+        }
+        input[type="radio"]{
+           margin: 0 10px 0 10px;
+        }
+
+        #question p {
+            font-size: large;
+        }
+    </style>
+</head>
+<body>
+    <nav class="navbar fixed-top navbar-light bg-light">
+        <ul class="navbar-nav justify-content-center">
+            <li class="nav-item">
+              <a class="nav-link" href="#">ABOUT</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Share</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">View</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">MyPage</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">LOGIN</a>
+            </li>
+          </ul>
+    </nav>
+    <div id="project-sign-up">
+        <p>Sign Up to AutSPACE</p>
+    </div>
+    <form id="form-check" action="{% url 'signup_frame4_test' %}" method="POST">
+        <div class="container">
+            <div id="question">
+                <p>Are you autistic?</p>
+            </div>
+            <div class="options">
+                <label for="explicit-label-name">YES </label>
+                <input type="radio" name="inlineRadioOptions" id="inlineRadio1" value="option1">
+                <label for="explicit-label-name">NO </label>
+                <input type="radio" name="inlineRadioOptions" id="inlineRadio2" value="option2">
+                <label for="explicit-label-name">NOT SURE </label>
+                <input type="radio" name="inlineRadioOptions" id="inlineRadio3" value="option3">
+                <label for="explicit-label-name">PREFER NOT TO SAY </label>
+                <input type="radio" name="inlineRadioOptions" id="inlineRadio4" value="option4">
+            </div>
+            <div class="submit">
+                <button type="button" class="btn btn-primary mb1 bg-greenyellow" form="form-check">CONTINUE</button>
```

Comment:
Clicking the submit button doesn't direct me to the next page. I'm not sure if the `form action` html attribute is supposed to create that effect, but you can sort of _cheat_ by surrounding `CONTINUE` with 
```html
<a href="/main/signup1">[TEXT]</a>
``` 

Of course, this won't send the form data to the server, but at the moment, it's not a concern for this current milestone. We're focused on establishing a flow, even if it's roundabout.

Reply:
Yes, I was facing the same issue. 

I have pushed the changes and updated it using the anchor tag thing. The caveat is that even without selecting and submitting the choices it user would go onto the next page by clicking on the submit button.